doc/source/serve/production-guide/index.md

```diff
@@ -0,0 +1,119 @@
+(serve-in-production)=
+
+# Production Guide
+
+The recommended way to run Ray Serve in production is on Kubernetes using the [KubeRay](kuberay-quickstart) [RayService] custom resource.
```

Comment:
The slightly awkward thing is that we currently consider this recommended way of running Serve to be alpha.
Should we express this fact here?

Reply:
Yeah it's a good point, let me soften the language and clarify...